packages/block-editor/src/layouts/flex.js

```diff
@@ -66,16 +66,19 @@ export default {
 		onChange,
 		layoutBlockSupport = {},
 	} ) {
-		const { allowOrientation = true } = layoutBlockSupport;
+		const { allowOrientation = true, allowJustification = true } =
```

Comment:
Just double checking the PR's description:

> This PR updates the controls so they are only output when allowJustification is true.

Is it right that `true` is the default?

Reply:
> Is it right that true is the default?

Yes!